docs/architecture/grpc-for-wcf-developers/introduction.md

```diff
@@ -1,11 +1,61 @@
 ---
 title: Introduction - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: Introduction
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Introduction
 
+Helping machines communicate with each other has been one of the primary preoccupations of the digital age. In particular, there is an ongoing effort to determine the optimal remote communication mechanism that will suit the interoperability demands of the current infrastructure. As you can imagine, that mechanism changes as either the demands or the infrastructure evolves.
+
+The release of .NET Core 3.0 marks a shift in the way that Microsoft delivers remote communication solutions to developers who want to deliver services across a range of platforms. .NET Core doesn't offer Windows Communication Foundation (WCF) out of the box but, with the release of version ASP.NET Core 3.0, it does provide built-in gRPC functionality.
```

Comment:
I think either here, or somewhere else in this introduction it is worth mentioning that gRPC is a popular framework in the wider programming community.

Basically I want to get the message across that gRPC wasn't picked randomly to be WCF's successor but because gRPC has mindshare.

Reply:
Good call, thanks.